wolvic/browser/wolvic_contents.cc

```diff
@@ -93,10 +93,12 @@ void WolvicContents::DidFinishLoad(content::RenderFrameHost* render_frame_host,
   webapps::InstallableManager* installable_manager =
       webapps::InstallableManager::FromWebContents(web_contents_.get());
 
-  webapps::InstallableParams params;
-  installable_manager->GetData(
-      params, base::BindOnce(&WolvicContents::OnDidGetInstallableData,
-                             weak_ptr_factory_.GetWeakPtr()));
+  if (installable_manager) {
```

Comment:
As we don't do anything else, you can just bail out
if (!installable_manager)
return;

Reply:
Right now, it is, but since this function is general, there is a possibility that other code will be inserted soon. That is why I did not use early return here.

Reply:
fair enough